cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -597,10 +597,11 @@ def test_vm_disk_reconfig_via_rest(appliance, full_vm):
     if not (BZ(1691635).blocks and full_vm.provider.one_of(RHEVMProvider)):
 
         # there will always be 2 disks after the disk has been added
-        disk_added = list(set(full_vm.configuration.disks) - set(initial_disks))[0]
+        disks_present = [disk.filename for disk in full_vm.configuration.disks]
```

Comment:
Can you add a comment as to why you have to use the disk.filename? If it's something obvious I'm missing, you can just respond here.